README.md

```diff
@@ -6,9 +6,25 @@ Check out [`fleetctl preview`](https://fleetdm.com/get-started) for a one-step s
 
 ### Development
 
+IMPORTANT: 
+* The `master` branch is used by `fleetctl` before version 4.5.0 and should not change anymore except for critical fixes.
```

Comment:
We can probably limit merges to this branch to zach or something like that, so nobody makes the mistake (except zach, I guess)